simtools/tests/test_telescope_model.py

```diff
@@ -100,11 +100,25 @@ def test_cfg_input():
     return
 
 
+def test_pars_from_db_handler():
```

Comment:
Should the test print something from the model? Not sure, just wondering.

Reply:
Yeah, this test is quite useless as it is. I was using it while implementing and didn't think of it as a real test. I think the best is to remove it actually. Anything that could be tested here will already be tested by other test functions.

Reply:
OK. I will approve the merge anyway now and you can remove it before or after the merge, whenever you want.